src/Geta.404Handler/Core/Web/HttpContextBaseExtensions.cs

```diff
@@ -28,5 +28,13 @@ public static HttpContextBase RedirectPermanent(this HttpContextBase context, st
             context.Response.RedirectPermanent(url, endResponse: false);
             return context;
         }
+
+        public static HttpContextBase RedirectTemporary(this HttpContextBase context, string url)
```

Comment:
Instead of a separate method, rename `RedirectPermanent` to just `Redirect` and pass `redirectType` as a parameter and move the redirect logic here.

Reply:
Corrected as per suggestion